cql3/expr/expression.hh

```diff
@@ -740,6 +740,9 @@ using single_column_restrictions_map = std::map<const column_definition*, expres
 // Extracts map of single column restrictions for each column from expression
 single_column_restrictions_map get_single_column_restrictions_map(const expression&);
 
+// Checks whether this expression isn't restricting any column
+bool is_empty_restriction(const expression&);
```

Comment:
Name could be better. No column != empty.

Reply:
Changed it so that it actually checks if the expression has anything apart from conjunctions.
Now the name is ok I think - `empty restriction` == `empty conjunction`.